andela/AndelaEatsUI
# #161079288 Implement suspend engagement functionality

#### What does this PR do?
- Implements `Delete Engagements` functionality.

#### Description of Task to be completed?
- Ensures an admin can delete an engagement

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git) and
-  cd into AndelaEatsUI and checkout to this branch `ft-admin-suspend-engagement-functionality-161079288`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Engagements`, and click on `Delete`
- run `npm test` to ensure all tests are passing


#### What are the relevant pivotal tracker stories?
- [##161079288](https://www.pivotaltracker.com/n/projects/2178749/stories/161079288)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1159" alt="screen shot 2018-11-15 at 4 07 13 pm" src="https://user-images.githubusercontent.com/21138774/48576733-9351d980-e915-11e8-8722-fe01fed4ef97.png">
